src/routes/api/storyboard/start/+server.ts

```diff
@@ -33,5 +36,18 @@ export const POST: RequestHandler = async ({ request }) => {
 		return json({ error: 'Failed to save storyboard' }, { status: 500 });
 	}
 
+	// Update user to include this storyboard
+	const user = await getUserById(userId);
```

Comment:
Can use `getUserFromEvent` instead to get user, like how how `/api/teams/create/` handles it.

`getUserFromEvent` goes through auth checks to ensure that it's a valid user. Relying on `userId` passed in body could possibly give a security risk that someone could be passing an invalid id to make use of it.